alvr/server_core/src/logging_backend.rs

```diff
@@ -18,10 +27,17 @@ pub fn init_logging(session_log_path: Option<PathBuf>, crash_log_path: Option<Pa
             let event_type = if maybe_event.starts_with('{') && maybe_event.ends_with('}') {
                 serde_json::from_str(&maybe_event).unwrap()
             } else {
-                EventType::Log(LogEntry {
-                    severity: LogSeverity::from_log_level(record.level()),
-                    content: message.to_string(),
-                })
+                let log_level = record.level();
+                if log_level <= LevelFilter::Info
+                    || alvr_common::filter_debug_groups(&maybe_event, &debug_groups_config)
```

Comment:
shouldn't this be a `&&`?

Reply:
This condition says that the log level must be <= info (Error + warn + info) or in case it's debug it needs to pass the groups filter. the groups filter will always allow any untagged log. The condition is made this way so that for error, warn and info are always allowed, you don't need to check the filter, so it's faster to compute.

Reply:
Oh log levels go that way

Reply:
yeah, I actually got it wrong the first time, and after testing i fixed it